app/webpacker/javascripts/jquery.autosize.js

```diff
@@ -0,0 +1,258 @@
+/*!
```

Comment:
Should we not bundle this as part of our yarn install process?

Reply:
Couldn't figure out upstream - I think really we should be dropping the jQuery code anyway - that can happen later though